confignode/src/main/java/org/apache/iotdb/confignode/consensus/ConsensusType.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.confignode.consensus;
+
+import java.util.Arrays;
+
+public enum ConsensusType {
```

Comment:
put this to consensus module?

Reply:
The consensus layer provides multiple implementations, but which consensus algorithm is needed should be decided when other modules instantiate consensus classes, so it seems odd to put this on consensus?

Reply:
I've disscued this with @OneSizeFitsQuorum. Maintain the ConsensusType in confignode currently is ok. We will optimize that later.